example/lib/main.dart

```diff
@@ -9,18 +14,42 @@ class MyApp extends StatefulWidget {
 }
 
 class _MyAppState extends State<MyApp> {
+  GlobalKey _globalKey = new GlobalKey();
+  bool isLoading = false;
   @override
   void initState() {
     super.initState();
   }
 
+  Future<Uint8List> _capturePng() async {
+    try {
+      RenderRepaintBoundary boundary =
+          _globalKey.currentContext.findRenderObject();
+      ui.Image image = await boundary.toImage(pixelRatio: 3.0);
+      ByteData byteData =
+          await image.toByteData(format: ui.ImageByteFormat.png);
+      final pngBytes = byteData.buffer.asUint8List();
+      return pngBytes;
+    } catch (e) {
+      print(e);
+      return Uint8List(0);
+    }
+  }
+
+  String emulationFor(String modelName) {
+    String emulation = 'StarGraphic';
+    if (modelName != null && modelName != '') {
+      final em = StarMicronicsUltilities.detectEmulation(modelName: modelName);
```

Comment:
Typo at StarMicronicsUltilities.
Should be StarMicronicsUtilities.

Reply:
Sure... I didn't notice it before sorry